pkgs/by-name/li/libjaylink/package.nix

```diff
@@ -37,7 +38,7 @@ stdenv.mkDerivation rec {
   '';
 
   postInstall = ''
-    install -Dm644 contrib/60-libjaylink.rules $out/lib/udev/rules.d/60-libjaylink.rules
+    install -Dm644 ../contrib/60-libjaylink.rules $out/lib/udev/rules.d/60-libjaylink.rules
```

Comment:
This is not a fixup? Is the change needed because meson builds out of source?

Reply:
Right, that's because of the switch to Meson.